pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -675,6 +799,515 @@ var _ = Describe("Topology", func() {
 			ExpectSkew(ctx, env.Client, "default", &topology[0]).To(ConsistOf(4, 4, 4))
 		})
 	})
+
+	Context("Pod Affinity", func() {
+		It("should schedule a pod with empty pod affinity and anti-affinity", func() {
+			Skip("enable after pod-affinity is finished")
+			ExpectCreated(ctx, env.Client)
+			pod := ExpectProvisioned(ctx, env.Client, selectionController, provisioners, provisioner, test.UnschedulablePod(test.PodOptions{
+				PodRequirements:     []v1.PodAffinityTerm{},
+				PodAntiRequirements: []v1.PodAffinityTerm{},
+			}))[0]
+			ExpectScheduled(ctx, env.Client, pod)
+		})
+		It("should respect pod affinity", func() {
+			Skip("enable after pod-affinity is finished")
```

Comment:
Can we put the skip directive at the context level?

Reply:
No, there was a feature request for that feature with ginkgo but it wasn't implemented.